File: 1st Class/Project Assignment/MonopolyDices.cpp
#include <cstdlib> 
#include <ctime>
#include <iostream> 
using namespace std; 

int main()
{
	srand(time(0));  ///random 
    

    while (true)
    {
        int x = 1+rand()%6; /// for the first dice=>random number between one and six
        int y = 1+rand()%6; ///for second dice=>random number between one and six
        if (x !=y){ ///If the value of the 1st dice and the 2nd dice are not the same, I want it to add up.
        	cout<<"Let's roll te dices!"<<endl;
        	cout<<"Dice1:"<<x<<"\nDice2:"<<y<<endl;
        	cout<<"Dice1+Dice2="<<x+y<<endl;
        	break;} ///döngüyü sonlandırma 
        else if (x==y){ ///if the value of the first dice and the second dice are the same I start a second loop in else if
        	cout<<"Let's roll te dices!"<<endl;
        	cout<<"Dice1:"<<x<<"\nDice2:"<<y<<"\nThe same number came from both dice.You have to roll again."<<endl;
        	cout<<"Rolling again..."<<endl;
        	int z = 1+rand()%6; 
			int k = 1+rand()%6; 
			if(z!=k){
				cout<<"Dice3:"<<z<<"\nDice4:"<<k<<endl;
				cout<<"Dice3+Dice4="<<z+k<<endl;
				break;}
			else if ( z==k ){
				cout<<"Dice3:"<<z<<"\nDice4:"<<k<<endl;
				cout<<"Same number rolled on both dice for the 2nd time..."<<endl;
				cout<<"Go to jail!";
				break;
			} break;
			
        	
		
		
    }
    return 0;
}}

File: 1stClass/Project Assignment/MonopolyDices.py
import random #for random number
import time   
while True: 
    x = random.randrange(1,7) #for 1st dice=>random number between one and six 
    y = random.randrange(1,7) #for 2nd dice=>random number between one and six
    if x != y: #If the value of the 1st dice and the 2nd dice are not the same, I want it to add up.
        print("Let's roll the dices!")
        time.sleep(1)  #i make it wait for 1 second before going to the next step
        print("Dice1:",x,"Dice2:",y)
        print("Dice1+Dice2=",x+y)
        break  #ending the loop
    elif x==y: #if the value of the first dice and the second dice are the same I start a second loop in elif
        print("Let's roll the dices!")
        time.sleep(1)
        print("Dice1:",x,"Dice2:",y,"\nThe same number came from both dice. You have to roll again.")
        time.sleep(1)
        print("Rolling again...")
        time.sleep(1)
        z = random.randrange(1,7) 
        k = random.randrange(1,7)
        if z != k:
               print("Dice3:",z,"Dice4:",k)
               print("Dice3+Dice4=",z+k)
               break
        elif z==k:
               print("Dice3:",z,"Dice4:",k)
               print("Same number rolled on both dice for the 2nd time...")
               time.sleep(1)
               print("Go to jail!")
               break
        break
